src/components/Header/AuthMenu.js

```diff
@@ -17,16 +27,27 @@ class AuthMenu extends Component {
   }
 
   render() {
+    const { showModal } = this.state;
+    let modal;
+    if (showModal) {
+      modal = <Signin toggleModal={this.toggleModal} />;
+    } else {
+      modal = null;
```

Comment:
I don't think you need the `else` statement in this condition. 
just assign modal to be `null` by default then 
```
let modal = null;
if(showModal) {
  modal = <Signin toggleModal={this.toggleModal} />;
}

Reply:
Thanks, I have done this